core/wiki/macros/sidebar-resizer.tid

```diff
@@ -0,0 +1,245 @@
+title: $:/core/procedures/sidebar-resizer
+
+\function convert.to.percentage(value) [<value>divide<widgetNodeWidth>multiply[100]]
+
+\function convert.to.percentage.value() [convert.to.percentage<value>]
+\function convert.to.cm.value() [convert.to.cm<value>]
+\function convert.to.mm.value() [convert.to.mm<value>]
+\function convert.to.Q.value() [convert.to.Q<value>]
+\function convert.to.in.value() [convert.to.in<value>]
+\function convert.to.pc.value() [convert.to.pc<value>]
+\function convert.to.pt.value() [convert.to.pt<value>]
+\function convert.to.em.value() [convert.to.em<value>]
+
+\function convert.to.pixels(value) [is.pixel.value<value>] [is.percentage.value<value>multiply<widgetNodeWidth>divide[100]] [is.cm.value<value>multiply[37.8]] [is.mm.value<value>multiply[37.8]divide[10]] [is.Q.value<value>multiply[37.8]divide[40]] [is.in.value<value>multiply[96]] [is.pc.value<value>multiply[96]divide[6]] [is.pt.value<value>multiply[96]divide[72]] [is.em.value<value>multiply{$:/themes/tiddlywiki/vanilla/metrics/fontsize}]
+
+\function convert.to.result(value,suffix) [<suffix>match[px]then<value>] [<suffix>match[%]then<convert.to.percentage.value>] [<suffix>match[cm]then<convert.to.cm.value>] [<suffix>match[mm]then<convert.to.mm.value>] [<suffix>match[Q]then<convert.to.Q.value>] [<suffix>match[in]then<convert.to.in.value>] [<suffix>match[pc]then<convert.to.pc.value>] [<suffix>match[pt]then<convert.to.pt.value>] [<suffix>match[em]then<convert.to.em.value>]
+
```

Comment:
It seems there is duplicated code. It looks very similar to: $:/core/functions/themes and